templates/cisco_ios_show_environment_temperature.template

```diff
@@ -0,0 +1,41 @@
+Value SWITCH (\d+)
+Value SWITCH_TEMPERATURE (\w+)
+Value INLET_TEMPERATURE_VALUE (\d+)
+Value INLET_TEMPERATURE_STATE (\w+)
+Value INLET_YELLOW_THRESHOLD (\d+)
+Value INLET_RED_THRESHOLD (\d+)
+Value HOTSPOT_TEMPERATURE_VALUE (\d+)
+Value HOTSPOT_TEMPERATURE_STATE (\w+)
+Value HOTSPOT_YELLOW_THRESHOLD (\d+)
+Value HOTSPOT_RED_THRESHOLD (\d+)
+
+Start
+  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$
+  ^Inlet\s+Temperature\s+Value:\s+${INLET_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$ -> Inlet
```

Comment:
```suggestion
  ^Inlet\s+Temperature\s+Value:\s+${INLET_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$ -> Inlet
  ^\s*$$
  ^. -> Error
```

Reply:
The error statement actually threw an error when evaluating Inlet Temperature Value. The value is repeated as a hotspot so I moved the line to Inlet and added error checking after each section... hard to explain see below:

Original
Start
^Switch\s+\d -> Continue.Record
^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$
^Inlet\s+Temperature\s+Value:\s+${INLET_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$ 
$ -> Inlet
  ^\s*$$
  ^. -> Error 
 
Inlet
  ^Temperature\s+State:\s+${INLET_TEMPERATURE_STATE}\s*$$
  ^Yellow\s+Threshold\s+:\s+${INLET_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
  ^Red\s+Threshold\s+:\s+${INLET_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Hotspot
  ^\s*$$
  ^. -> Error 
 
Hotspot
  ^Hotspot\s+Temperature\s+Value:\s+${HOTSPOT_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$
  ^Temperature\s+State:\s+${HOTSPOT_TEMPERATURE_STATE}\s*$$
  ^Yellow\s+Threshold\s+:\s+${HOTSPOT_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
  ^Red\s+Threshold\s+:\s+${HOTSPOT_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Record Start
  ^\s*$$
  ^. -> Error 


Now
Start
  ^Switch\s+\d -> Continue.Record
  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$ -> Inlet
  ^\s*$$
  ^. -> Error 
 
Inlet
  ^Inlet\s+Temperature\s+Value:\s+${INLET_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$ 
  ^Temperature\s+State:\s+${INLET_TEMPERATURE_STATE}\s*$$
  ^Yellow\s+Threshold\s+:\s+${INLET_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
  ^Red\s+Threshold\s+:\s+${INLET_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Hotspot
  ^\s*$$
  ^. -> Error 
 
Hotspot
  ^Hotspot\s+Temperature\s+Value:\s+${HOTSPOT_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$
  ^Temperature\s+State:\s+${HOTSPOT_TEMPERATURE_STATE}\s*$$
  ^Yellow\s+Threshold\s+:\s+${HOTSPOT_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
  ^Red\s+Threshold\s+:\s+${HOTSPOT_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Record Start
  ^\s*$$
  ^. -> Error